llpc/test/shaderdb/relocatable_shaders/TriangleVs_CheckNoteSectionForCacheHash.spvasm

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Add a comment describing what the test is suppose to check.

Reply:
done